index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Quiz</title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+  <article id="Widgets">
```

Comment:
There should be an `<article>` for each card. Also, this tag is not closed, or at least it did not register as closed in my brackets plugin.